mock-interactions.js

```diff
@@ -188,13 +244,14 @@
    * @param {HTMLElement} node The node to fire the event on.
    * @param {Object} xy Optional. The (x,y) coordinates the mouse event should be fired from.
```

Comment:
This annotation seems wrong. in previous iterations it was missing, in this iteration options does not have to do with xy state.